File: openapi/code_samples/python/airtime/get_all_commissions/code.py
import requests
import json

url = "https://topups.reloadly.com/operators/commissions"

payload={}
headers = {
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'Accept': 'application/com.reloadly.topups-v1+json',
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)

File: openapi/code_samples/nodejs/airtime/get_commission_by_id/code.js
var myHeaders = new Headers();
myHeaders.append("Authorization", "Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");
myHeaders.append("Accept", "application/com.reloadly.topups-v1+json");

var requestOptions = {
  method: 'GET',
  headers: myHeaders,
  redirect: 'follow'
};

fetch("https://topups.reloadly.com/operators/173/commissions", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));